### PR TITLE
Fix js implicite conversion of data value

### PR DESCRIPTION
**Pull request by natacha-beck**

### CODE REVIEW

**Comment by prioux** [comment]
What about all the other JS files ?

**Comment by prioux** [comment]
OK thin it covers all

**Comment by natacha-beck** [comment]
I have `grep` on the whole code. It should be ok.

File: BrainPortal/public/javascripts/cbrain.js [hunk 1]
```diff
@@ -303,7 +303,7 @@
           if (!error_message) {
             error_message = "<span class='loading_message'>Error loading element</span>";
           }
-          if (replace === "true") {
+          if (replace === true) {
```
Comment (prioux) [inline BrainPortal/public/javascripts/cbrain.js]: should we change this to

```
if (replace) {
```

File: BrainPortal/public/javascripts/cbrain.js [hunk 2]
```diff
@@ -324,7 +324,7 @@
       var scroll_bottom   = current_element.data("scroll-bottom");
 
       if (data) data = jQuery.parseJSON(data);
-      if (scroll_bottom === "false") scroll_bottom = false;
+      if (scroll_bottom === false) scroll_bottom = false;
```
Comment (prioux) [inline BrainPortal/public/javascripts/cbrain.js]: What's this junk?

File: BrainPortal/public/javascripts/cbrain.js [hunk 3]
```diff
@@ -772,7 +772,7 @@
 
       data[submit_name] = submit_value;
 
-      if (ajax_submit !== "false") {
+      if (ajax_submit !== false) {
```
Comment (prioux) [inline BrainPortal/public/javascripts/cbrain.js]: The same here,

```
if (ajax_submit) {
```

**Comment by prioux** [comment]
Nat, you've tried it right, and the `replace` and `invert` tests work properly ? If so I will merge.

**Comment by prioux** [comment]
I attempted a squash merge with Natacha's permission, just to see what would happen.

**Pull request merged**